ownership-chain/runtime/laos/src/configs/transaction_payment.rs

```diff
@@ -0,0 +1,21 @@
+use super::fee::ToAuthor;
+use crate::{Balance, Balances, Runtime, RuntimeEvent, WeightToFee, MICROUNIT};
+use frame_support::{parameter_types, weights::ConstantMultiplier};
+use polkadot_runtime_common::SlowAdjustingFeeUpdate;
+
+parameter_types! {
+	/// Relay Chain `TransactionByteFee` / 10
+	pub const TransactionByteFee: Balance = 10 * MICROUNIT; // TODO check this value with the one used in deposit function
+	pub const OperationalFeeMultiplier: u8 = 5;
+}
+
+// TODO: following has to be checked with this: https://github.com/moonbeam-foundation/moonbeam/blob/31dd0f3703d844b139a2f0fafde91025a8b97771/runtime/moonbeam/src/lib.rs#L361
```

Comment:
missing TODO

Reply:
https://github.com/freeverseio/laos/issues/424 added to a story in the backlog